localstack/cli/lpm.py

```diff
@@ -59,6 +60,7 @@ def install(package, parallel):
     """
     console.print(f"resolving packages: {package}")
     installers: Dict[str, Callable] = InstallerManager().get_installers()
+    config.dirs.mkdirs()
```

Comment:
Is this necessary? It seems to be done implicitly when loading the `config.py` for most cases.